frontend/src/components/common/chip/styles.module.scss

```diff
@@ -0,0 +1,20 @@
+.chip {
+  min-height: 40px;
+  margin-right: 10px;
+  padding: 10px;
```

Comment:
I'd make vertical padding +-5px, and horizontal - 10px. Currently Chip component is too large in comparison to other table elements